### DISCUSSION TITLE
Link utensils to steps like ingredients

### DISCUSSION THREAD

**Discussion by niklas152** [mealie-recipes/mealie#4086] [Q&A] [answered]
Hi there, first of all thanks for this great tool!
The importer works great for me and I love the UI both on desktop and on mobile.
Especially like that you can link ingredients to steps, and the display of that in cook mode. Now my question is, maybe I have just missed it, can we do the same for utensils? I.e., link the ones required to a step to that step and have them displayed in a similar fashion in cook mode?

Thanks and kind regards
Niklas

**Comment by michael-genson** [comment, marked as answer]
Currently this isn't possible, the tool is tied to the recipe, but not to the step. Certainly worth a PR, though!
